src/PlanetParade.vue

```diff
@@ -327,13 +360,13 @@
           <v-window-item>
             <v-card class="no-bottom-border-radius scrollable">
               <v-card-text class="info-text no-bottom-border-radius">
-                <h4>What is the planet parade?</h4>
+                <h3>What is the planet parade?</h3>
                 <p>
-                  During February 2025, seven planets&#8212;Mercury, Saturn, Neptune, Venus, Uranus, Jupiter, and Mars&#8212;will be visible in the night sky all at once!
+                  During February 2025, seven planets&#8212;Mercury, Saturn, Neptune, Venus, Uranus, Jupiter, and Mars&#8212;will be visible in the night sky all at once! From a reasonably dark, clear sky, you should be able to see five of the seven planets by eye.
                 </p> 
-                <h4>Cool! How do I see it?</h4>
+                <h3>Cool! How do I see the planet parade?</h3>
                 <p>
-                  The best time to see it will be shortly after it gets dark, maybe 20-30 minutes after sunset. In the Northern Hemisphere, you'll want a clear view towards the southern half of the sky, from west to east. 
+                  The best time to see it will be at the end of February, shortly after it gets dark, maybe 20-30 minutes after sunset. In the Northern Hemisphere, you'll want a clear view towards the southern half of the sky, from west to east. 
```

Comment:
All of February you can see this. It is just late February that we can see Mercury. I think we want to encourage people to look now, but I think saying late February will make people wait.

Reply:
Very good point - I will edit.